src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/InternalComponentsMapper.java

```diff
@@ -240,9 +236,9 @@ private DocumentBuilder getDocumentBuilder() throws ParserConfigurationException
         return documentBuilderFactory.newDocumentBuilder();
     }
 
-    private String putMainImageReferenceInBodyNode(Node bodyNode, String mainImage) throws TransformerException {
-        Element newElement = bodyNode.getOwnerDocument().createElement("content");
-        newElement.setAttribute("id", mainImage);
+    private String putMainImageReferenceInBodyNode(Node bodyNode, String mainImageUUID) throws TransformerException {
+        Element newElement = bodyNode.getOwnerDocument().createElement("ft-content");
+        newElement.setAttribute("url", "http://api.ft.com/content/" + mainImageUUID);
```

Comment:
So as it not really a trivial fix, I'll make an improvement task for the dynamic url generation based on if it is a test article (test.api.ft.com) or not. But will extract the string in a static field.